tutorials/audio.py

```diff
@@ -0,0 +1,94 @@
+"""
+    mss.tutorials.audio
+    ~~~~~~~~~~~~~~~~~~~
+
+    This python script is meant for generating audio of our choice from text files describing the tutorial.
+
+    This file is part of mss.
+
+    :copyright: Copyright 2021 Hrithik Kumar Verma
+    :copyright: Copyright 2021 by the mss team, see AUTHORS.
+    :license: APACHE-2.0, see LICENSE for details.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+"""
+import os
+import time
+import playsound
+import googletrans as gt
+from gtts import gTTS
```

Comment:
on linux we could also git flite a try

`flite -f tutorial_waypoints.txt -voice slt`

Reply:
> on linux we could also try flite
> 
> `flite -f tutorial_waypoints.txt -voice slt -o example.wav`
> https://anaconda.org/anaconda/flite
> 
> or for other languages also pico2wave
> e.g.
> `pico2wave --lang en-US --wave us.wav "This is the waypoints tutorial,i.e., whenever we are going to plan a flight track, we have to place the waypoints in some places and, form a flight path. Now, we will learn the basic technacalities of the waypoint. Here is the MSS(mission support system) Software and it will be maximised automatically in few seconds."`

Yes, we could try it later!